friends/src/Components/modalClass.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Button, 
+    Modal,
+    ModalHeader,
+    ModalBody,
+    ModalFooter ,
+    Form,
+    FormGroup,
+    Input,
```

Comment:
The Modal Component is awesome! Awesome job implementing it in more than one place. Looks great